SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.ImportBefore.targets

```diff
@@ -13,8 +13,7 @@
 
     <SonarQubeTempPath Condition=" $(SonarQubeBuildDirectory) != '' AND Exists('$(SonarQubeBuildDirectory)\.sonarqube\conf\SonarQubeAnalysisConfig.xml') ">$(SonarQubeBuildDirectory)\.sonarqube</SonarQubeTempPath>
 
-    <AutoImportSonarQubeAnalysisTargets Condition=" $(SonarQubeTempPath) != '' ">true</AutoImportSonarQubeAnalysisTargets>
```

Comment:
Miscellaneous - this property is not used so I've deleted it and the two tests that checked it.